tests/configlet/util/common.py

```diff
@@ -225,6 +233,8 @@ def cmp_dump(db_name, orig_db_dir, clet_db_dir):
     for k in diff:
         log_error("{}: Missing key: {}".format(fname, k))
         mismatch_cnt += 1
+        if not msg:
+            msg = "Missing key: {}".format(k)
```

Comment:
This is *only* for reporting in assert's error message.
Instead of just saying "_failed to compare dumps_" which does not give any hint, now we say "_failed to compare: <the first error encountered>_", to give little more context.

This additional context will be helpful. Being assert message, it can't be too long.